### DISCUSSION TITLE
windows support

### DISCUSSION THREAD

**Discussion by raspberrypisig** [pop-os/libcosmic#489] [General]
Unsure the right place to ask.

Basically this issue, but for me came across on windows

https://github.com/edfloreshz/cosmic-app-template/issues/7

Since the book refers to the template, thought I'd ask here.

**Comment by mkrueger** [comment]
I'm interested in building libcosmic applications. I've a few egui based applications but with libcosmic they would fit more into my Pop!OS Desktop :).

Howver most of my users are Windows/OSX so cross platform is a deal breaker for me. Would help to make it easier to build the examples on other platforms as well automatically on github. So far I had no luck running libcosmic samples on windows :(.

Didn't try osx yet.
